modules/monitoring-creating-cross-project-alerting-rules-for-user-defined-projects.adoc

```diff
@@ -81,7 +83,7 @@ spec:
 <1> Ensure that you specify the project that you defined in the `namespacesWithoutLabelEnforcement` field.
 <2> The name of the alerting rule you want to create.
 <3> The duration for which the condition should be true before an alert is fired.
-<4> The PromQL query expression that defines the new rule.
+<4> The PromQL query expression that defines the new rule. Use the expression to filter which projects are included or excluded from the alerting rule.
```

Comment:
I am afraid we don't need to change it
the prometheus expression in the doc is only one example, and it's exclude projects `openshift|kube).*|default`, user can define any expr.
`expr: kube_namespace_labels{namespace!~"(openshift|kube).*|default",label_pod_security_kubernetes_io_enforce!="restricted"} `